ca_on_toronto/__init__.py

```diff
@@ -1,22 +1,49 @@
+from utils import CanadianJurisdiction
 from opencivicdata.divisions import Division
 from pupa.scrape import Organization
 
-from utils import CanadianJurisdiction
-
 
 class Toronto(CanadianJurisdiction):
-    classification = "legislature"
-    division_id = "ocd-division/country:ca/csd:3520005"
-    division_name = "Toronto"
-    name = "Toronto City Council"
-    url = "http://www.toronto.ca"
+    classification = 'executive'  # just to avoid clash
```

Comment:
No need to change this - there's no possible clash.

Reply:
Without classification I get:

 `Unable to import Jurisdiction subclass from ca_on_toronto. Jurisdiction subclass may be missing a division_id or classification.`

Reply:
I mean you can leave it as "legislature"